backend/src/routes/v1/approvals.ts

```diff
@@ -212,3 +236,14 @@ export const postApprovalResponse = [
     })
   },
 ]
+async function getManagerAndReviewer(metadata: ModelMetadata) {
```

Comment:
This should not be a route, but should instead probably be in the schema service?